.ado/jobs/node-tests.yml

```diff
@@ -11,7 +11,7 @@ parameters:
 
   - name: versions
     type: object
-    default: [14, 16]
+    default: [16]
```

Comment:
We might want to add node 18 in here since it's been LTS for a long time now

Reply:
Node.js is about to release version 20.

Reply:
We should definitely add 20 when it's out and see if there's any problems as new devs will probably install 20 by default

Reply:
Addition of 18 threw errors in the testing. Looks like there's an API that is no longer available on 18 that we use. I'll file an issue to track addressing that.

Reply:
You know, that sounds familliar - something that changed at like 18.5 or something.